resources/tests.mjs

```diff
@@ -1,5 +1,7 @@
+import{ BenchmarkTestStep} from "./benchmark-runner.mjs";
```

Comment:
Nit: There should be a space between `BenchmarkTestStep` and `}`.

Reply:
DONE.